modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -44,9 +44,15 @@ See the table below for detailed information:
 | tcnative version
 | Direct link
 
+| 5.9
+| 4.1.92.Final
```

Comment:
just fyi In current dev it's already changed: 

        <version>4.1.94.Final</version>